woocommerce/woocommerce-gutenberg-products-block
# Create single product block placeholder

_#2399 is a mammoth PR so I'm breaking out some fixes and changes to review separately._

Creates a placeholder block/component for the Single Products Block.

- Register the block and add the webpack config
- Add icon/name/description
- Adds an icon for the not yet implemented reset button
- Keep the block behind the experimental feature flag
- Simple e2e test for insertion

### How to test the changes in this Pull Request:

1. Do a development build
2. Ensure the Single Product Block shows up in the inserter with an icon
3. Thats it! The block does nothing and has no interface at this point. Everything else is for followups.
